geoMean.js

```diff
@@ -0,0 +1,11 @@
+// Napisz funkcję geoMean(), która zwróci średnią geometryczną podanej tablicy.
+
+function geoMean(array) {
+  return Math.pow(array.reduce(function(prev, next) {
```

Comment:
Everything is ok except one parameter name.

According to this:

https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce

the first parameter is not a previous element. It's an accumulator.